generator/generate.ts

```diff
@@ -100,7 +100,7 @@ async function handleGeneratedSchema(readme: string, schemaPath: string, autoGen
 }
 
 async function execAutoRest(tmpFolder: string, params: string[]) {
-    await executeCmd(__dirname, autorestBinary, params);
+    await executeCmd(__dirname, `${__dirname}/node_modules/.bin/${autorestBinary}`, params);
```

Comment:
>${__dirname}/node_modules/.bin/${autorestBinary} [](http://example.com/codeflow?start=33&length=48)

I remember having issues in the past with platform specific path delimiters, not sure if makes sense to use [path.join](https://nodejs.org/api/path.html#path_path_join_paths)